### ISSUE TITLE
2.11 night mode will change some templates' css?

### ISSUE THREAD

**Issue by XiaoNiaoZZJ** [ankidroid/Anki-Android#6392]
I found my template be changed by ankidroid.
Is it a new feature to offer a more universal night mode？
How can I disable this feature.It makes my template display incorrectly,and I don't know how to fix it.
The template works well at ankimobile , ankiweb and anki on windows 10.

![image](https://user-images.githubusercontent.com/13731650/83960350-78113b00-a8ba-11ea-9741-6d5525233354.png)

**Comment by david-allison-1** [comment]
Hi @XiaoNiaoZZJ,

You can find instructions here:

https://docs.ankiweb.net/#/templates/styling?id=platform-specific-css

If you want to target AnkiDroid specifically, use `.android {}`

Thanks for using AnkiDroid,

David

**Comment by XiaoNiaoZZJ** [comment]
> Hi @XiaoNiaoZZJ,
> 
> You can find instructions here:
> 
> https://docs.ankiweb.net/#/templates/styling?id=platform-specific-css
> 
> If you want to target AnkiDroid specifically, use `.android {}`
> 
> Thanks for using AnkiDroid,
> 
> David

Thanks for your suggestion.

I have downgraded my AnkiDroid to 2.10.3 already and it displays well again. 

It seems that my compatibility javascript to detect night mode and change some classname cause the problem.

I suggest use a independent css to control night mode.Changing users' css may cause some compatibility issues such as  some compatibility css will also be changed.I feel sorry about I can't verify if AnkiDroid will also change my ".android ... {css}" because I have downgraded already when I received your comment.

I have solved the problem. Thank you.

xnzzj.
